Assignment-2/KaprekarConstant.py

```diff
@@ -0,0 +1,25 @@
+def takeInputFromUser():
```

Comment:
The current name implies that it's merely retrieving input, but it's also validating the input length. A more appropriate name might be "getValidInputFromUser" to convey both actions.